mockdata/api/clusters_mgmt/v1/clusters/21696acc0dbkvh0mh4lranlmvvqb11lg/node_pools.json

```diff
@@ -26,7 +26,8 @@
         }
       },
       "availability_zone": "us-east-1d",
-      "subnet": "subnet-0da631ca7d1b2ef9c"
+      "subnet": "subnet-0da631ca7d1b2ef9c",
+      "imageType": "Windows"
```

Comment:
This addition propagates a Machine Type which is Windows LI compatible, meaning it can enable Windows LI (as agreed with the backend team)